server/templates/custom_dc/unsdg_staging/base.html

```diff
@@ -0,0 +1,125 @@
+{#-
```

Comment:
how much this file can share with the unsdg/base.html? Might be helpful to make one inherit another so changes can happen just in one place.